src/syscall.c

```diff
@@ -52,15 +52,18 @@ const struct syscall_list syscall_lists[] = {
 #endif
 #ifdef AUDIT_ARCH_AARCH64
     SYSCALL_LIST(AUDIT_ARCH_AARCH64, aarch64),
+    SYSCALL_LIST(AUDIT_ARCH_ARM, arm),
```

Comment:
From what I see, AArch64 syscall list is not duplicating arm one because the extract.py script dumps syscall table and there is separate syscall table fir every arch within kernel.

Reply:
I am not sure `AUDIT_ARCH_AARCH64` implies `AUDIT_ARCH_ARM`: https://elixir.bootlin.com/linux/latest/source/include/uapi/linux/audit.h#L389

Or do you want me to do:

```c
#if defined(AUDIT_ARCH_AARCH64)
    SYSCALL_LIST(AUDIT_ARCH_AARCH64, aarch64),
    SYSCALL_LIST(AUDIT_ARCH_ARM, arm),
#elif defined(AUDIT_ARCH_ARM)
    SYSCALL_LIST(AUDIT_ARCH_ARM, arm),
#elif defined(AUDIT_ARCH_X86_64)
… 
#else
#error
#endif
```

here?

Reply:
I mean if `AUDIT_ARCH_ARM` is not defined while `AUDIT_ARCH_AARCH64` is defined then the above will not compile anyhow.

Reply:
Uhm, I think I got what I did I do wrong here :+1: My intention was to check if `KAFEL_DEFAULT_TARGET_ARCH` contains the respective architecture to include what is only needed. Target `i386` can not be executed on `aarch64` stuff etc. I will checnge to `#ifdef __x86-64__` etc here.